kore/src/Kore/Rewrite/Function/Evaluator.hs

```diff
@@ -332,12 +332,7 @@ reevaluateFunctions ::
     -- | Function evaluation result.
     Pattern RewritingVariableName ->
     simplifier (OrPattern RewritingVariableName)
-reevaluateFunctions sideCondition rewriting = do
-    let (rewritingTerm, rewritingCondition) = Pattern.splitTerm rewriting
-    OrPattern.observeAllT $ do
-        simplifiedTerm <- simplifyConditionalTerm sideCondition rewritingTerm
-        simplifyCondition sideCondition $
-            Pattern.andCondition simplifiedTerm rewritingCondition
+reevaluateFunctions = simplifyPattern
```

Comment:
Do you think it's useful to keep `reevaluateFunctions` since it's just another name for `simplifyPattern`?